packages/editor-ui/src/components/canvas/Canvas.vue

```diff
@@ -143,15 +133,16 @@ const disableKeyBindings = computed(() => !props.keyBindings);
  * @see https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#whitespace_keys
  */
 
-const panningKeyCode = ' ';
-const isPanningEnabled = ref(false);
+const isPanning = ref(false);
+const panningKeyCode = ref<string[]>([' ', controlKeyCode]);
+const panningMouseButton = ref<number[]>([1]);
 const selectionKeyCode = ref<true | null>(true);
 
-onKeyDown(panningKeyCode, () => {
+onKeyDown(panningKeyCode.value, () => {
 	setPanningEnabled(true);
```

Comment:
@alexgrozav do we still need this `setPanningEnabled`?

Reply:
Nope, good point. Removed it.